conradomateu/day08/day08Tests.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import conradomateu.day08.day08 as solver
+
+
+class MyTestCase(unittest.TestCase):
+    @classmethod
```

Comment:
I didn't know about this

Reply:
It is very useful , I did not know too. I will use It in the nexts problems.

Reply:
And what does this do? Just remove the `self` parameter in the method?

Reply:
Is like a Set Up for the class to reuse code.